stf/feature_engineering/feature_free_days.py

```diff
@@ -44,27 +46,31 @@ def create_holiday_functions():
                 consist of "Is" + the_name_of_the_holiday_to_be_checked
     """
 
-    # Manully generated csv including all dutch holidays for different regions
-    df_holidays = pd.read_csv(HOLIDAY_CSV_PATH, index_col=None)
-    df_holidays["datum"] = pd.to_datetime(df_holidays.datum).apply(lambda x: x.date())
+    if years is None:
+        years = [2020, 2021]
```

Comment:
Don't use hard coded years here. If you think we need a default option its probably better to use the current year. Something along the line of:

```python
if years is None:
  now = datetime.now()
  years = [now.year, now.year + 1]
```